src/components/volunteer/VolunteerFormCalendar.tsx

```diff
@@ -399,6 +408,52 @@ export const VolunteerFormCalendar = ({
   );
 };
 
+const fileFilters = (
+  fileRegex: any,
+  calendarData: {
+    id: number;
+    content: { files: any[] };
+  }
+) =>
+  JSON.stringify(
+    calendarData?.content?.files
+      .map(({ file_name: infoText, mime_type: mimeType, url: uri, id: fileId }: any) => {
+        const isFile = fileRegex.test(infoText);
+
+        if (isFile) return { entryId: calendarData?.id, fileId, infoText, mimeType, uri };
+        else return;
+      })
+      .filter((otherFiles: any) => otherFiles != null)
+  );
+
+const filerParseAndUpload = (calendarNewData: VolunteerCalendar, id: number) => {
+  const images = jsonParser(calendarNewData.images);
+  const documents = jsonParser(calendarNewData.documents);
+  const uris: { uri: StringConstructor; mimeType: StringConstructor }[] = [];
```

Comment:
I really don't remember how and why I did this. Yes, I'm not good with types, but I know that a `string` will come here instead of `StringConstructor`. 🤷‍♂️